governance-exercises/Le Bateleur - Poll Exercise 02/Course of Action.md

```diff
@@ -0,0 +1,61 @@
+
+But a determination from the Governance Facilitator is not to be the final word. Per [Atlas 2.6.6.1.1](https://mips.makerdao.com/mips/details/MIP101#2-6-6-1-1), a review of the decision by all FacilitatorDAOs is required. Because FacilitatorDAOs don't yet exist, running a governance poll can serve as a crucial "check" on the Governance Facilitator's power. There is also (outdated) [historical support](https://forum.makerdao.com/t/mip102c2-sp2-mip-amendment-subproposals/20016/19#h-1011-constitutional-boundaries-of-the-arbitration-scope-framework-67) for using a governance poll to adjudicate derecognition matters. (The former Maker Constitution, 10.11.3.4, states, "During the Pregame, all actions of derecognition must be decided through a MKR governance poll...")
+
+---
+
+### Should a whistleblower bounty be paid to Alice?
+
+This cannot be answered until the final determination is made as to whether Clive's identity has been compromised. [Atlas 2.6.6](https://mips.makerdao.com/mips/details/MIP101#2-6-6-aligned-delegate-privacy) states that a whistleblower bounty is paid when an actor has "provided useful information for determining that the privacy of an AD was compromised." (Arguably, there is a bit of ambiguity here, but not so much as to preclude a reasonable interpretation.) I interpret this to mean that a determination has to be made that the privacy of the Aligned Delegate was in fact compromised *before* a whistleblower is eligible to receive a bounty. 
+
+Therefore, Alice's bounty cannot be included in the coming Monday's poll concerning the breach-of-anonymity evidence against Clive. Should that poll determine the materials provided rise to the level of "significant suspicion", thus triggering Clive's derecognition, I would then do the necessary follow up with Alice on the bounty.
+
+---
+
+### *Epilogue*
+
+Had I started from ground zero in the Exercise, I would not have run a poll. But I think Atlas 2.6.6.1.1 provides fairly viable support for running one. 
+
+Given my mandate as Governance Facilitator, I believe this situation would also call for follow-up. I would consider taking proactive steps to resolve the ambiguity in the current rules, so that similar situations in the future can be more reliably adjudicated in an aligned manner. The ethical concerns in play are a huge factor in my thinking.
```

Comment:
Good. Fixing only the immediate problem is not the best outcome here. You always need to be thinking about how to adjust the rules and processes to fix the same problem in the future, and even potential variations on the problem.